tox.ini

```diff
@@ -42,6 +42,7 @@ commands=
     rm -f {envdir}/src/sideboard/plugins/uber
 
 [testenv:flake8]
+basepython = python3
```

Comment:
When I ran ``tox`` on my Vagrant box, I got some errors which I tracked down as indicating that ``flake8`` wasn't correctly parsing Python 3 syntax, and this line tells Tox to use the Python 3 version of ``flake8``, which caused it to start working.